hack/cherry_pick_pull.sh

```diff
@@ -168,6 +170,9 @@ for pull in "${PULLS[@]}"; do
       fi
     done
 
+    # remove the patch file from /tmp
+    rm -f "/tmp/${pull}.patch"
```

Comment:
oh yeah i was just going to clean up even if there were conflicts but i will move to the `return_to_kansas` instead thats a better idea